website/blog/2019-08-31-dandelion-overview.md

```diff
@@ -1,68 +1,75 @@
+The fundamental mechanism that defines MolochDAO is the ability for members to exit the organization with a proportional share of the organization's assets, so long as they have not recently voted to approve a pending proposal. This creates a *loose partnership* where members can exit before a bad or otherwise controversial decision is made, and because of that guarantee there is less friction for collaboration in the first place.
 
-Moloch was developed with a philosophy of minimizing complexity and code footprint. The organization does not try and do everything, but what it does do it does very well. In contrast, Aragon has been built with a philosophy of modularity and code-reuse. As a result there is a growing library of useful, audited, off-the-shelf components that can be combined to create and expand a software-defined organization. After reviewing the [specification](https://github.com/MolochVentures/moloch) we determined that there were some key features of Moloch that we wanted to replicate in order to enable Aragon organization to adopt similar mechanisms, but stay true to the Aragon development philosophy of creating modular and re-usable components.
+MolochDAO was developed with a philosophy of minimizing complexity and code footprint. The organization does not try and do everything, but what it does do it does very well. In contrast, Aragon has been built with a philosophy of modularity and code-reuse. As a result there is a growing library of useful, audited, off-the-shelf components that can be combined to create and expand a software-defined organization. After reviewing the [specification](https://github.com/MolochDAOVentures/MolochDAO) we determined that there were some key features of MolochDAO that we wanted to replicate in order to enable Aragon organization to adopt similar mechanisms, but stay true to the Aragon development philosophy of creating modular and re-usable components.
 
 Specifically we wanted to make sure that we met the following requirements:
 
 1) Members should be able to redeem tokens for a proportional share of the organizations assets
 2) It should be possible to restrict exit actions based on whether the individual had voted yes in a pending proposal.
 3) New members should be able to propose a payment to the organization in exchange for the organizations tokens
-4) Proposals should require the proposer to lock tokens in order to ensure each proposal has been well considered by the proposer.
+4) Proposals should require the proposer to lock tokens in order to ensure that the incentives of voters are aligned with their votes
```

Comment:
I don't think this is what the locking mechanism in either Dandelion or MolochDAO does... The locking mechanism in both is mostly to prevent spam. There is no slashing that occurs if the proposal is approved/rejected, so I would say we should leave this as it was before, or talk specifically about the need to impose a cost on proposals to prevent spam.

Reply:
Wouldn't locking the token prevent someone from voting for something that damages the org, then redeeming their tokens and not reaping the consequences of their vote?

Reply:
Also the term "well considered" is vague. We should replace those words with something more descriptive